tests/scripts/pulp_deb/test_deb_remote.sh

```diff
@@ -72,5 +72,14 @@ expect_succ pulp deb remote update --name "${ENTITIES_NAME}" --architecture ""
 expect_succ pulp deb remote show --name "${ENTITIES_NAME}"
 assert "$(echo "$OUTPUT" | jq -r .architectures)" == "null"
 
+#  test with a gpg key if cli_version >= 0.24.0
+cli_version=$( pulp  --version  | awk '{print $NF}' )
+if [ "$({  echo $cli_version ; echo "0.24.0"; } | sort -V | tail -n1)" = "$cli_version" ]
```

Comment:
So you are saying we could add `pulp debug has-cli [--name common]`? Not a bad idea.

Reply:
yes, for that specific case 
The failure seems only linked to a cli version not a specific version of one of integrated module (deb/rpm/core)

A similar option to the has-plugin would have simplify the version check

The has-plugin contains everything needed to easily  run this kind of test
Options:
  --name TEXT         [required]
  --min-version TEXT  Succeed only if the installed version is not smaller.
  --max-version TEXT  Succeed only if the installed version is smaller.
  --specifier TEXT    Succeed only if the installed version is contained.
  --help              Show this message and exit.

I saw the build failure, the fix is easy ,I will modify that and resubmit when I got home
 (missing double quote)

Reply:
the build failure should be fixed

Reply:
https://github.com/pulp/pulp-cli/pull/941
This is what you are looking for, i believe. It won't make it into any of the old CLI versions however. Sorry.